pkg/infra/remotecache/remotecache.go

```diff
@@ -72,12 +86,12 @@ type CacheStorage interface {
 
 	// Count returns the number of items in the cache.
 	// Optionaly a prefix can be provided to only count items with that prefix
+	// DO NOT USE. Not available for memcached.
```

Comment:
Would it be a problem to remove count from the interface and wherever it’s used, use a temporary interface to check if it’s implemented?

```go
type countable interface {
	Count(ctx context.Context, prefix string) (int64, error)
}

func fun() {
…
if counter, countable := remoteCache; countable {
county, err := counter.Count(countext, "monte-cristo")
…
}
…
}
```

Reply:
:+1: Seems good to me @sakjur , would make it safer even